pages/trainers_en.html

```diff
@@ -11,10 +11,13 @@ <h2>Melanie Ahn, 2nd Dan holder</h2>
 	Melanie mentions: “Taekwondo boosted my self-confidence, and gave me many new friends, so I hope you will find as much joy and growth in Taekwondo as I did”
 </p>
 
-<img src="images/trainers/ab.jpg" alt="Abdenasser Eddeane" width=589/>
-<h2>Abdenasser Eddeane, 2nd Dan holder</h2>
+<img src="images/trainers/tung.jpg" alt="Le Xuan Tung" width=589/>
+<h2>Le Xuan Tung, 2nd Dan holder</h2>
 <p>
-	Abdenasser Eddeane, 2nd Dan holder. Ab has Trainers License Level 3, which is recognised by the NOC / NSF, the Dutch Olympic Committee.
-	Ab started Taekwondo when he was 10 years old and reached his 1st  Dan at the age of 13. One year later, at the young age of 14, he reached his 2nd DAN.
-	Ab loves Taekwondo, and is especially passioned about sparring. He has participated in national and international tournaments. At Ilyeo he will be responsible for boosting the club’s sparring level, in both physical and technical aspects. Ab is in charge of Friday trainings at Ilyeo.
+	I started Taekwondo at Ilyeo from 2011, reached 1st Dan in 2016 and 2nd Dan in 2018.
+	I love Taekwondo because of its strong, quick and efficient movements.
+	Tung is in charge of the monday and friday trainings at Ilyeo
```

Comment:
Dagen van de week zijn met hoofdletters in het Engels

Reply:
Zal ik aanpassen